core/src/main/java/tc/oc/pgm/command/MapCommand.java

```diff
@@ -81,6 +83,12 @@ public void maps(
       search = search.filter(map -> matchesName(map, name));
     }
 
+    if (phase != null) {
+      search = search.filter(map -> matchesPhase(map, phase));
+    } else {
+      search = search.filter(map -> map.getPhase() == Phase.PRODUCTION);
+    }
```

Comment:
Can be simplified to:

```java
Phase phase = phaseStr == null ? Phase.PRODUCTION : Phase.of(query);
if (phase == null) throw ...; // invalid phase passed as parameter
search = search.filter(map -> map.getPhase() == phase);
```

This also removes the matchesPhase method calling Phase.of(query); for each of the maps

Reply:
Thanks for the feedback @Pablete1234! Excellent suggestions 👍